frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -70,13 +70,21 @@ export const TARGET_AREA_TO_NAME = {
     'posthog-3000': 'PostHog 3000',
 }
 
+export const SEVERITY_LEVEL_TO_NAME = {
+    critical: 'Critical - Outage / Data loss / Breach',
```

Comment:
Sentence case these, from:

> Critical - Outage / Data loss / Breach

to 

> Critical - Outage / data loss / breach

etc

Reply:
```suggestion
    critical: 'Critical - Outage / data loss / breach',
```